packages/animation_metadata/pubspec.yaml

```diff
@@ -1,6 +1,5 @@
 name: animation_metadata
 description: A package that just defines how metadata for an animation is held.
-author: Chromium Authors
```

Comment:
Is this just getting rid of a vestigial tie to the original source?

Reply:
Yes.  I removed it because of a new lint that says the author field is deprecated.